tests/utils/got.js

```diff
@@ -0,0 +1,16 @@
+const got = require('got')
+
+const TIMEOUT = 1e4
+
+// Default got retry status code with the addition of 403
+// eslint-disable-next-line no-magic-numbers
+const STATUS_CODE = [403, 408, 413, 429, 500, 502, 503, 504, 521, 522, 524]
```

Comment:
Some of the deploy tests are occasionally failing with 403:
https://github.com/netlify/cli/runs/1696253114?check_suite_focus=true#step:7:366

I'm not sure yet if this is an issue with our tests because of https://github.com/netlify/cli/pull/1733/files#r557434973.

I would rather to stabilize the tests first then look into it.